data/mimeType.go

```diff
@@ -0,0 +1,461 @@
+package data
+
+// CODE GENERATED AUTOMATICALLY WITH gopkg.in/src-d/enry.v1/internal/code-generator
+// THIS FILE SHOULD NOT BE EDITED BY HAND
+// Extracted from github/linguist commit: d5c8db3fb91963c4b2762ca2ea2ff7cfac109f68
+
+var LanguagesMime = map[string]string{
+	"1C Enterprise":    "",
+	"ABAP":             "",
+	"ABNF":             "",
+	"AGS Script":       "text/x-c&#43;&#43;src",
```

Comment:
This string is escaped, you must use text/template instead html/template as import in the file where the function to generate this file is defined